elements/rh-icon/docs/30-code.md

```diff
@@ -0,0 +1,24 @@
+{% renderInstall lightdomcss=true %}{% endrenderInstall %}
+
+## Usage
+
+<rh-code-block>
+<script type="text/html"><rh-icon icon="alert"></rh-icon></script>
+</rh-code-block>
+
+### Loading
+
+Icons load lazily by default, meaning that the browser will not attempt to fetch and render the icon until it scrolls into view. You can change this with the loading attribute, which has three values:
+
+1. `lazy` (the default): load icons when they scroll into view
+2. `idle`: load each icon on the page as soon as the browser enters an idle state Or, on less-capable browsers, at the next frame
+3. `eager`: each icon will begin to load and render the moment it connects to the DOM.
+
+You might choose to enable eager rendering for "above-the-fold" content, but keep lazy loading for the rest of the page.
+
+<rh-code-block>
+<script type="text/html"><rh-icon icon="alert" loading="eager"></rh-icon></script>
+</rh-code-block>
+
+{% renderCodeDocs hideDescription=true %}{% endrenderCodeDocs %}
+
```

Comment:
> ## ~~Labeling~~ Labelling Icons
>
> If an icon is interactive or otherwise not merely decorative, it ~~will need to~~ must be labeled using `aria-label` or `aria-labelledby`.

Reply:
Any non-presentational element must have an `aria-label` _AND_ `role="img"`. Axe will throw an error if an `rh-icon` has an `aria-label` but no `role`.

Reply:
Any non-presentational element must have an `aria-label` _AND_ `role="img"`. Axe will throw an error if an `rh-icon` has an `aria-label` but no `role`.

The a11y docs under "Labelling icons" should reflect this.